### DISCUSSION TITLE
付费数天收不到激活码

### DISCUSSION THREAD

**Discussion by kessty** [getlantern/lantern-forum-cn#610] [Q&A]
@Mike777Khoo 购买lantern已经好几天了，到现在也没收到激活码，也没任何反馈。就这服务真是无语了，赶紧退款吧，邮箱：pcfan@163.com 

![WechatIMG16](https://github.com/getlantern/lantern-forum-cn/assets/173220908/16cb39b2-3248-44a4-bcd1-ab45e1aceebd)

**Comment by MaxMKMax** [comment]
@kessty 您好. 我这里可以找负责人帮您弄激活码. 您先别退款, 等一等我们好吗? 谢谢您!

**Comment by MaxMKMax** [comment]
@kessty https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Comment by kessty** [comment]
资料都有 你直接处理吧，如果还这么折腾那我就退款了。

| |
张伟
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年06月21日 02:48 |
| 收件人 | getlantern/lantern-forum-cn ***@***.***> |
| 抄送人 | kessty ***@***.***>,
Mention ***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 付费数天收不到激活码 (Discussion #610) |

@kesstyhttps://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Reply by MaxMKMax** [reply]
@kessty 您好. 请问您有在链接里提交资料和问题吗? 很抱歉让您久等了. 周二肯定帮您搞定. 您也留意看有没有收到客服的邮件. 估计他们昨天又给您发激活码了. 如果没有, 我周一帮您追问. 谢谢您的等待!

**Comment by MaxMKMax** [comment]
@kessty 您好. 技术团队说您不需要激活码了. 只需要连接上您的设备就可以了.
如何连接上: https://lantern.io/zh-Hans/faq

**Comment by kessty** [comment]
你好，我这边到现在也没有激活成功，，一直提交不了，客户端也是最新的版本。
















在 2024-06-25 01:12:15，"Mike K" ***@***.***> 写道：

@kessty 您好. 技术团队说您不需要激活码了. 只需要连接上您的设备就可以了.
如何连接上: https://lantern.io/zh-Hans/faq
